mono/eglib/gmodule.h

```diff
@@ -20,6 +20,9 @@ typedef enum {
 	G_MODULE_BIND_LOCAL = 0x02,
 	G_MODULE_BIND_MASK = 0x03
 } GModuleFlags;
+
+G_ENUM_FUNCTIONS (GModuleFlags)
```

Comment:
I wonder if each of the uses of `G_ENUM_FUNCTIONS` should come with a "fixme" note.